src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemingViewsFeatureProvider.cs

```diff
@@ -16,7 +16,7 @@ public class ThemingViewsFeatureProvider : IApplicationFeatureProvider<ViewsFeat
     {
         public static readonly string ThemeLayoutFileName = $"DefaultOrchardCoreThemingLayout{RazorViewEngine.ViewExtension}";
         public static readonly string ThemeLayoutFilePath = $"/{ThemeLayoutFileName}";
-        public static readonly string MvcViewDocumentKind = $@"""{MvcViewDocumentClassifierPass.MvcViewDocumentKind}""";
+        public static readonly string MvcViewDocumentKind = @"""MvcViewDocumentClassifierPass.MvcViewDocumentKind""";
```

Comment:
That's why in the original comment I did put the namespace in the string, and ensure this string is in a package that is referenced by these compiled views.

Reply:
Okay I understand, I will do ;)